src/azure-cli/azure/cli/command_modules/feedback/custom.py

```diff
@@ -552,10 +557,35 @@ def _get_parent_proc_name():
     logger.debug("Total minified issue length is %s", len(minified_issue_body))
     logger.debug("Total formatted url length is %s", len(formatted_issues_url))
 
-    return _ISSUES_TEMPLATE_PREFIX.format(pretty_url_name), formatted_issues_url, original_issue_body
+    return _ISSUES_TEMPLATE_PREFIX.format(pretty_url_name, _CLI_ISSUES_URL, _EXTENSIONS_ISSUES_URL), \
+        formatted_issues_url, original_issue_body
 
 
-def _get_minified_issue_url(command_log_file, format_dict, is_ext, capacity):
+def _get_extension_repo_url(ext_name, raw=False):
+    _NEW_ISSUES_STR = '/issues/new'
+    try:
+        project_url = resolve_project_url_from_index(extension_name=ext_name)
+        if _is_valid_github_project_url(url=project_url):
+            raw_url = project_url.lower().strip('/') + _NEW_ISSUES_STR
+            # Prettify the url for cli extensions repo
+            if not raw and raw_url == _RAW_EXTENSIONS_ISSUES_URL:
+                return _EXTENSIONS_ISSUES_URL
+            return raw_url
+    except (CLIError, NoExtensionCandidatesError) as ex:
+        # since this is going to feedback let it fall back to the generic extensions repo
+        logger.debug(ex)
+    return _RAW_EXTENSIONS_ISSUES_URL if raw else _EXTENSIONS_ISSUES_URL
+
+
+def _is_valid_github_project_url(url):
```

Comment:
@adewaleo I looked at the current project urls in the index file and found a few links like..  https://github.com/Azure/azure-cli-extensions/tree/master/src/azure-firewall which would have broken in case of my previous check. 
Making this change to strictly verify the project url for format **https://github.com/org/project** format.

Reply:
Where you able to manually test your changes. I.e. running az feedback on:
- an azure firewall command (this special case)
- a regular extension command with its own project url
- a regular extension command with our (cli extension) project url
- regular cli command?

Just test that you can run -h with each command and open the browser to a reasonable page with the correct content.

Thanks a lot!

Reply:
Yes, tested with-
1. an azure firewall command - **az network firewall list**
2. a regular extension command with its own project url - **az devops project list** 
3. a regular extension command with our (cli extension) project url - **az alias list**
4. regular cli command - **az extension add**
5. File generic issue option - **az feedback - option 0**
6. Extension with non giithub.com url in project home page  - **az ml workspace list**

Reply:
Note that -h commands are not logged as extension commands in the metadata so they will come to azure-cli repo irrespective of my change. 
Not sure if this is a recent regression or has been like this always but this behavior can be verified in 2.0.67.

Reply:
@adewaleo I am done with code, test and manual verification from my side. Please take care of the review and merge.

Reply:
The behavior has been like this